UnattendResources/Logon.ps1

```diff
@@ -344,45 +345,78 @@ try
     if ($vmwareToolsPath) {
         Install-VMwareTools
     }
-    Run-CustomScript "RunBeforeCloudbaseInitInstall.ps1"
-    $Host.UI.RawUI.WindowTitle = "Installing Cloudbase-Init..."
 
-    $programFilesDir = $ENV:ProgramFiles
+    $unattendedXmlPath = $null
 
-    $CloudbaseInitMsiPath = "$resourcesDir\CloudbaseInit.msi"
-    $CloudbaseInitMsiLog = "$resourcesDir\CloudbaseInit.log"
+    if ($installCloudbaseInit){
+        Run-CustomScript "RunBeforeCloudbaseInitInstall.ps1"
+        $Host.UI.RawUI.WindowTitle = "Installing Cloudbase-Init..."
 
-    if (!$serialPortName) {
-        $serialPorts = Get-WmiObject Win32_SerialPort
-        if ($serialPorts) {
+        $programFilesDir = $ENV:ProgramFiles
+
+        $CloudbaseInitMsiPath = "$resourcesDir\CloudbaseInit.msi"
+        $CloudbaseInitMsiLog = "$resourcesDir\CloudbaseInit.log"
+
+        if (!$serialPortName) {
+            $serialPorts = Get-WmiObject Win32_SerialPort
+            if ($serialPorts) {
             $serialPortName = $serialPorts[0].DeviceID
+            }
         }
-    }
 
-    $msiexecArgumentList = "/i $CloudbaseInitMsiPath /qn /l*v $CloudbaseInitMsiLog"
-    if ($serialPortName) {
-        $msiexecArgumentList += " LOGGINGSERIALPORTNAME=$serialPortName"
-    }
+        $msiexecArgumentList = "/i $CloudbaseInitMsiPath /qn /l*v $CloudbaseInitMsiLog"
+        if ($serialPortName) {
+            $msiexecArgumentList += " LOGGINGSERIALPORTNAME=$serialPortName"

```

Comment:
are these tabs? if yes, replace them with 4 spaces

Reply:
I think those were tabs and he replaced them with spaces, that's why they show up like this. 
Sublime shows them as spaces to me.